packages/hub/scripts/start-ganache.js

```diff
@@ -0,0 +1,19 @@
+process.env.BABEL_ENV = "development";
```

Comment:
Could these `start-ganache` scripts be consolidated into one script in `devtool`s? Or did you run into issues with that?